### ISSUE TITLE
Testsuite doesn't check regressions

### ISSUE THREAD

**Issue by dolmen** [360EntSecGroup-Skylar/excelize#150]
The testsuite uses almost only `t.Log` instead of validating assertions and failing with [`t.Error`](https://golang.org/pkg/testing/#T.Error), [`t.Fatal`](https://golang.org/pkg/testing/#T.Fatal) or [`t.Fail`](https://golang.org/pkg/testing/#T.Error):
```
$ grep -E -c 'Error|Fatal|Fail|panic' *_test.go
excelize_test.go:3
sheet_test.go:0
```

This means that only code coverage is verified, but **regressions are not checked automatically** (humans have to read the `go test -v` output and have to know what is expected: that's not helpful for external contributors).

**Comment by dolmen** [comment]
It seems that most `if err != nil` blocks in `*_test.go` files that call `t.Log` could call `t.Fatal` instead.

Before:
```go
if err  != nil {
    t.Log(err)
}
```
After:
```go
if err  != nil {
    t.Fatal(err)
}
```

**Comment by dolmen** [comment]
@xuri Would you accept a patch that fixes this issue in the way I suggest in [previous comment](#issuecomment-343431642)?

**Comment by xuri** [comment]
This is a late reply, this issue resolved by #322.
